### PR TITLE
fix: add eventVersion check for map tasks

### PR DESCRIPTION
**Pull request by olga-union**
Signed-off-by: Olga Nad <olga@union.ai>

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Added event version check to show new map task UI (graph and details tab) only for events with version >= 1.

## Screenshots
With eventVersion
Screen Shot 2022-05-18 at 4 54 54 PM
Without eventVersion
Screen Shot 2022-05-18 at 4 58 40 PM

## Tracking Issue
fixes https://github.com/flyteorg/flyteconsole/issues/381

## Follow-up issue
_NA_

### CODE REVIEW

**Review by anrusina — DISMISSED** [review]
LGTM

File: packages/zapp/console/src/components/Executions/TaskExecutionsList/TaskExecutionsList.tsx [hunk 1]
```diff
@@ -42,9 +42,11 @@ export const TaskExecutionsListContent: React.FC<{
   return (
     <>
       {taskExecutions.map((taskExecution) => {
-        const taskType = taskExecution.closure.taskType ?? undefined;
+        const {
+          closure: { taskType, metadata, eventVersion = 0 },
+        } = taskExecution;
         const useNewMapTaskView =
-          isMapTaskType(taskType) && taskExecution.closure.metadata?.externalResources;
+          isMapTaskType(taskType ?? undefined) && eventVersion >= 1 && metadata?.externalResources;
```
[the remark this answers is not in the document]
Comment (olga-union) [reply packages/zapp/console/src/components/Executions/TaskExecutionsList/TaskExecutionsList.tsx:49]: good idea!

[the remark this answers is not in the document]
Comment (olga-union) [reply packages/zapp/console/src/components/Executions/TaskExecutionsList/TaskExecutionsList.tsx]: added helper and tests

**Review by anrusina — APPROVED** [review]
LGTM 

# 🚀

**Pull request merged**